docs/design/devices.md

```diff
@@ -0,0 +1,34 @@
+# K8s support "/sys/fs/cgroup/devices"
```

Comment:
What you said make sense, but I think different devices could have difference implementation in kubelet, absolutely, and this time I only want to enable the NVIDIA GPU. But regarding the changes in scheduler, should have an unified interface for all the devices. It kind of like enable all the devices by scheduler's perspective. So considering all changes in kubelet and scheduler to make GPU enabled, what should be the name for the changes? Do you have any ideas about it?